editor/src/components/inspector/inspector-common.tsx

```diff
@@ -73,3 +86,24 @@ export function filterKeepFlexContainers(
     MetadataUtils.isFlexLayoutedContainer(MetadataUtils.findElementByElementPath(metadata, e)),
   )
 }
+
+export function detectFlexDirection(
+  metadata: ElementInstanceMetadataMap,
+  elementPath: ElementPath,
+): FlexDirection {
+  const element = MetadataUtils.findElementByElementPath(metadata, elementPath)
+  if (element == null) {
+    return 'row'
+  }
+
+  return stringToFlexDirection(element.computedStyle?.['flexDirection'] ?? null) ?? 'row'
+}
+
+export const isFlexColumn = (flexDirection: FlexDirection): boolean =>
+  flexDirection.startsWith('column')
+
+export const metadataSelector = (store: EditorStorePatched): ElementInstanceMetadataMap =>
+  store.editor.jsxMetadata
+
+export const selectedViewsSelector = (store: EditorStorePatched): ElementPath[] =>
+  store.editor.selectedViews
```

Comment:
These should probably be in a file for common selectors rather than Inspector specific ones

Reply:
moved!